runtime/onert/core/include/backend/ITensorRegistry.h

```diff
@@ -59,6 +59,13 @@ struct ITensorRegistry
    * @return false if not supported
    */
   virtual bool setMigrantTensor(const ir::OperandIndex &, IPortableTensor *) { return false; }
+  /**
+   * @brief Get the tensor that is associated with the given index
```

Comment:
```suggestion
   * @brief Check if a tensor with the given index exists
```

Reply:
Thanks for your pointing out. I will fix it.